pkg/sdk/parameters_impl.go

```diff
@@ -199,85 +172,123 @@ func GetSessionParametersUnsetFrom(params map[string]any) (*SessionParametersUns
+		unsetField = &sessionParametersUnset.StrictJSONOutput
 	case SessionParameterTimestampDayIsAlways24h:
-		sessionParametersUnset.TimestampDayIsAlways24h = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampDayIsAlways24h
 	case SessionParameterTimestampInputFormat:
-		sessionParametersUnset.TimestampInputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampInputFormat
 	case SessionParameterTimestampLTZOutputFormat:
-		sessionParametersUnset.TimestampLTZOutputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampLTZOutputFormat
 	case SessionParameterTimestampNTZOutputFormat:
-		sessionParametersUnset.TimestampNTZOutputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampNTZOutputFormat
 	case SessionParameterTimestampOutputFormat:
-		sessionParametersUnset.TimestampOutputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampOutputFormat
 	case SessionParameterTimestampTypeMapping:
-		sessionParametersUnset.TimestampTypeMapping = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampTypeMapping
 	case SessionParameterTimestampTZOutputFormat:
-		sessionParametersUnset.TimestampTZOutputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimestampTZOutputFormat
 	case SessionParameterTimezone:
-		sessionParametersUnset.Timezone = Bool(true)
+		unsetField = &sessionParametersUnset.Timezone
 	case SessionParameterTimeInputFormat:
-		sessionParametersUnset.TimeInputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimeInputFormat
 	case SessionParameterTimeOutputFormat:
-		sessionParametersUnset.TimeOutputFormat = Bool(true)
+		unsetField = &sessionParametersUnset.TimeOutputFormat
+	case SessionParameterTraceLevel:
+		unsetField = &sessionParametersUnset.TraceLevel
+	case SessionParameterTransactionAbortOnError:
+		unsetField = &sessionParametersUnset.TransactionAbortOnError
 	case SessionParameterTransactionDefaultIsolationLevel:
-		sessionParametersUnset.TransactionDefaultIsolationLevel = Bool(true)
+		unsetField = &sessionParametersUnset.TransactionDefaultIsolationLevel
 	case SessionParameterTwoDigitCenturyStart:
-		sessionParametersUnset.TwoDigitCenturyStart = Bool(true)
+		unsetField = &sessionParametersUnset.TwoDigitCenturyStart
 	case SessionParameterUnsupportedDDLAction:
-		sessionParametersUnset.UnsupportedDDLAction = Bool(true)
+		unsetField = &sessionParametersUnset.UnsupportedDDLAction
 	case SessionParameterUseCachedResult:
-		sessionParametersUnset.UseCachedResult = Bool(true)
+		unsetField = &sessionParametersUnset.UseCachedResult
 	case SessionParameterWeekOfYearPolicy:
-		sessionParametersUnset.WeekOfYearPolicy = Bool(true)
+		unsetField = &sessionParametersUnset.WeekOfYearPolicy
 	case SessionParameterWeekStart:
-		sessionParametersUnset.WeekStart = Bool(true)
+		unsetField = &sessionParametersUnset.WeekStart
 	default:
 		return fmt.Errorf("%s session parameter is not supported", string(parameter))
 	}
+	*unsetField = Bool(true)
```

Comment:
Had to pause for a minute to figure out you're assigning a setter that you set at the end 😅